src/gui/mainwindow.cpp

```diff
@@ -679,7 +679,9 @@ void MainWindow::displayRSSTab(bool enable)
         if (!m_rssWidget) {
             m_rssWidget = new RSSWidget(m_tabs);
             connect(m_rssWidget.data(), &RSSWidget::unreadCountUpdated, this, &MainWindow::handleRSSUnreadCountUpdated);
-#ifndef Q_OS_MAC
+#ifdef Q_OS_MAC
+            m_tabs->addTab(m_rssWidget, tr("RSS (%1)").arg(RSS::Session::instance()->rootFolder()->unreadCount()));
```

Comment:
I wonder how "lupdate" handles duplicate strings?
Maybe it would be better to use something like:
```c++
const int indexTab = m_tabs->addTab(m_rssWidget, tr("RSS (%1)").arg(RSS::Session::instance()->rootFolder()->unreadCount()));
#ifdef Q_OS_MAC
Q_UNUSED(indexTab);
#else
m_tabs->setTabIcon(indexTab, GuiIconProvider::instance()->getIcon("application-rss+xml"));
#endif
```

Reply:
I checked and the translations for that string seem to be picked up correctly on macOS.